CONTRIBUTING.md

```diff
@@ -175,33 +175,42 @@ git clone https://github.com/source-repo/existing-plugins.git
 git clone https://github.com/backstage/community-plugins.git
 ```
 
-2. Identify the plugin(s) you wish to migrate. If you're migrating multiple plugins, is recommended to group the migration of these by workspace.
+2. Identify the plugin(s) you wish to migrate. If you're migrating multiple plugins, it's recommended to group the migration of these by workspace.
 
 3. Within the `backstage/community-plugins` repository create a new branch for your changes:
 
 ```sh
 git checkout -b migrate-workspace
 ```
 
-3. Create a new workspace in the community plugins repository.
+4. Create a new workspace in the community plugins repository. See the [Creating a new Workspace](#creating-a-new-workspace) section for more information.
 
-4. Copy the plugin files from the source repository to the `backstage/community-plugins` repository.
+5. Copy the plugin files from the source repository to the `backstage/community-plugins` repository.
 
 ```sh
 cp -r ../existing-plugins/plugins/plugin-name plugins/
 ```
 
-5. Ensure all metadata files (`package.json`) are updated to reflect the new repository. This includes updating repository URLs, issues URLs, and other references.
+6. Ensure all metadata files (`package.json`) are updated to reflect the new repository. This includes updating repository URLs, issues URLs, and other references.
 
-6. Add maintainers to the `CODEOWNERS` file for the new workspace.
+7. Generate API reports for the new plugin(s) in the workspace. See the [API Reports](#api-reports) section for more information.
+
+8. Add required Backstage metadata to the `package.json` file for the new plugin(s) in the workspace. This can usually be resolved by running the command below.
```

Comment:
If I'm not mistaken any new plugin should be getting this added already, this isn't a requirement that is specific to this repo. We might want to reword this to be along the lines of: "you may need to do this ..."

Reply:
I've updated this section to be a bit more explicit about migrating _older_ plugins. I think the wording is okay with it being under the **Manual migration steps** section.